pkg/util/kubeobjects/object_test.go

```diff
@@ -0,0 +1,25 @@
+package kubeobjects
+
+import (
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"testing"
+)
+
+func TestKey(t *testing.T) {
+	type args struct {
+		object client.Object
+	}
+	tests := []struct {
+		name string
+		args args
+		want client.ObjectKey
+	}{
+		{name: "tttttt", args: args{object: nil}, want: client.ObjectKey{}},
```

Comment:
```suggestion
		{name: "handle nil value", args: args{object: nil}, want: client.ObjectKey{}},
```

I don't really understand the test, but it kinda looks like a nil check test, would this name not be better?

Reply:
fixed in f984e317